tests/unit/twitter.js

```diff
@@ -4,13 +4,14 @@
   var deps = [
     'assert',
     'nock',
+    'underscore',
```

Comment:
Haven't heard of lodash. What do we use?

On Saturday, April 18, 2015, Anthony Van de Gejuchte <
notifications@github.com> wrote:

> In tests/unit/twitter.js
> <https://github.com/botwillacceptanything/botwillacceptanything/pull/403#discussion_r28647644>
> :
>
> > @@ -4,13 +4,14 @@
> >    var deps = [
> >      'assert',
> >      'nock',
> > +    'underscore',
>
> Should be lodash?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/botwillacceptanything/botwillacceptanything/pull/403/files#r28647644>
> .
>


-- 
----------------------------------------------------------------
Shale Craig
shalecraig@gmail.com